.github/workflows/streaming-tests.yml

```
@@ -15,23 +19,24 @@ jobs:
         with:
           update-conda: true
           python-version: ${{ matrix.python-version }}
-          conda-channels: conda-forge
       - uses: actions/checkout@v3
       - run: git fetch --prune --unshallow --tags
 
       - name: Install pytest
-        run: pip install pytest
+        run: |
+          pip install pytest
+          pip install pytest-cov
 
       - name: Install package
         run: pip install ".[dandi]"
-      - name: Install HDMF-zarr # temporary
-        run: pip install hdmf-zarr
-      - name: Install latest HDMF # temporary
-        run: pip install -U hdmf
-      - name: Uninstall h5py
-        run: pip uninstall -y h5py
-      - name: Install ROS3
-        run: conda install -c conda-forge h5py
 
-      - name: Run pytest on stream tests
-        run: pytest tests/ -rsx tests/streaming_tests.py
+      - name: Run pytest on stream tests with coverage
+        run: pytest -rsx --cov=nwbinspector --cov-report xml:./coverage.xml tests/streaming_tests.py
+
```

```suggestion
      - name: Run pytest on stream tests with coverage
        run: pytest -rsx --cov=nwbinspector --cov-report xml:./coverage.xml tests/streaming_tests.py

      - name: Run pytest on stream cli tests with coverage
        run: pytest -rsx --cov=nwbinspector --cov-report xml:./coverage.xml tests/streaming_cli_tests.py
```

I couldn't tell if the cli tests were getting run anywhere since they got separated out. You can ignore this suggestion if they're being run elsewhere.

---

Another good catch - fixed